ludwig/schema/features/preprocessing/set.py

```diff
@@ -63,7 +63,7 @@ class SetPreprocessingConfig(BasePreprocessingConfig):
 
 @DeveloperAPI
 @register_preprocessor("set_output")
-@dataclass(repr=False)
+@dataclass(repr=False, order=True)
 class SetOutputPreprocessingConfig(SetPreprocessingConfig):
```

Comment:
Should Set/TextOutputPreprocessingConfig inherit from `SequenceOutputPreprocessingConfig`?

Reply:
We could have `TextOuptutPreprocessingConfig` inherit from `SequenceOutputPreprocessingConfig` (the defaults are a little too different for set for it to make sense IMO. However, I would say that if we do want to do this, we would want the text input feature preprocessing config to inherit from sequence as well and there looks to be one default discrepancy so I think it would warrant it's own PR. Personally, although we can inherit here, I think there's value in a verbose list here for a couple reasons. **A)** it preserves the order of the schema - inheritance ends up tacking any overridden parameters on the end which causes the order to be unideal sometimes. **B)**  I think that for documentation purposes, I find it kind of convenient to be able to project search a specific default without having to redirect to a file where the defaults are inherited. If we think it's valuable enough I'm happy to change, but my vote would be to keep them separated. WDYT?